src/utils/webhooks.py

```
@@ -0,0 +1,50 @@
+    if not urls:
+        return
+
+    for url in urls:
+        try:
+            response = requests.post(url, json=payload, timeout=timeout)
+            logger.info(
+                "webhook: sent | url=%s status=%s",
+                url,
+                response.status_code,
+            )
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)